Dockerfile.pilot-cassandra

```diff
@@ -1,7 +1,12 @@
 FROM alpine:3.6
 
 ADD http://search.maven.org/remotecontent?filepath=org/jolokia/jolokia-jvm/1.4.0/jolokia-jvm-1.4.0-agent.jar /jolokia.jar
+ADD https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.3.0/jmx_prometheus_javaagent-0.3.0.jar /jmx_prometheus_javaagent.jar
+ADD https://raw.githubusercontent.com/prometheus/jmx_exporter/master/example_configs/cassandra.yml /jmx_prometheus_javaagent.yaml
```

Comment:
Could we pull this from a sha (0b490c14b6a8b53518b63aaaf02bf769e2eada4e is latest) instead of master?